<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vigenere</class>
 <widget class="QWidget" name="Vigenere">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>–®–∏—Ñ—Ä –í–∏–∂–µ–Ω–µ—Ä–µ–∞</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QFrame" name="Frame1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="3" column="0">
       <widget class="QPlainTextEdit" name="MessageEdit"/>
      </item>
      <item row="5" column="0" rowspan="2">
       <widget class="QPlainTextEdit" name="CodeEdit"/>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="EncodeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>–ó–∞—à–∏—Ñ—Ä–æ–≤–∞—Ç—å —Å–æ–æ–±—â–µ–Ω–∏–µ</string>
        </property>
        <property name="text">
         <string>ü†ì</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="DecodeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>–†–∞—Å—à–∏—Ñ—Ä–æ–≤–∞—Ç—å –∫—Ä–∏–ø—Ç–æ–≥—Ä–∞–º–º—É</string>
        </property>
        <property name="text">
         <string>ü†ë</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="KeyEdit"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="CodeLabel">
        <property name="text">
         <string>–ö—Ä–∏–ø—Ç–æ–≥—Ä–∞–º–º–∞</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="MessageLabel">
        <property name="text">
         <string>–°–æ–æ–±—â–µ–Ω–∏–µ</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="KeyLabel">
        <property name="text">
         <string>–ö–ª—é—á</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="Frame2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="0" rowspan="2" colspan="2">
       <widget class="QListWidget" name="AlphabetsList">
        <property name="toolTip">
         <string>–í—ã–±–µ—Ä–∏—Ç–µ –æ–¥–∏–Ω –∞–ª—Ñ–∞–≤–∏—Ç</string>
        </property>
        <property name="currentRow">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>abcdefghijklmnopqrstuvwxyz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ABCDEFGHIJKLMNOPQRSTUVWXYZ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>–∞–±–≤–≥–¥–µ—ë–∂–∑–∏–π–∫–ª–º–Ω–æ–ø—Ä—Å—Ç—É—Ñ—Ö—Ü—á—à—â—ä—ã—å—ç—é—è</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>–ê–ë–í–ì–î–ï–Å–ñ–ó–ò–ô–ö–õ–ú–ù–û–ü–†–°–¢–£–§–•–¶–ß–®–©–™–´–¨–≠–Æ–Ø</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="AlphabetsLabel">
        <property name="text">
         <string>–ê–ª—Ñ–∞–≤–∏—Ç—ã</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="AddAlphabetButton">
        <property name="text">
         <string>–î–æ–±–∞–≤–∏—Ç—å</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
